PowerEditor/src/Parameters.h

```diff
@@ -838,6 +838,7 @@ struct NppGUI final
 	bool _monospacedFontFindDlg = false;
 	bool _findDlgAlwaysVisible = false;
 	bool _confirmReplaceInAllOpenDocs = true;
+	bool _muteSounds = true;

```

Comment:
Should I make it "false" by default?

Reply:
Yes, please.

Reply:
I think the general idea is that existing behavior should be the default, so that is why this should be `false`